ovh/resource_cloud_project_gateway_test.go

```diff
@@ -0,0 +1,78 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+var testAccCloudProjectGatewayConfig = `
+resource "ovh_cloud_project_network_private" "mypriv" {
+	service_name  = "%s"
+  vlan_id       = "%d"
+  name          = "%s"
+  regions       = ["%s"]
+}
+
+resource "ovh_cloud_project_network_private_subnet" "myprivsub" {
+  service_name  = ovh_cloud_project_network_private.mypriv.service_name
+  network_id    = ovh_cloud_project_network_private.mypriv.id
+	region        = "%s"
```

Comment:
```suggestion
  region        = "%s"
```

Reply:
do not appears on my IDE, strange 
i've manged to reedit and just make tab to be sure